nextcord/application_command.py

```diff
@@ -92,6 +93,13 @@
 else:
     _CustomTypingMetaBase = object
 
+if sys.version_info >= (3, 10):
```

Comment:
I think `utils.PY_310` can be used for this. I just came across it and it looks like it's exactly this condition but in a constant.